cargo-shuttle/src/lib.rs

```diff
@@ -377,4 +428,131 @@ mod tests {
             path_from_workspace_root("examples/axum/hello-world/")
         );
     }
+
+    mod check_lib_version {
+        use super::*;
+
+        #[test]
+        fn invalid_manifests() {
+            Shuttle::check_lib_version(Document::new(), &Version::parse("0.4.1").unwrap())
+                .unwrap_err();
```

Comment:
For all the `unwrap_err()` tests, maybe we should check the error is what we expect?